src/open_inwoner/pdc/views.py

```diff
@@ -202,12 +213,17 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ProductFormView(BaseBreadcrumbMixin, CategoryBreadcrumbMixin, DetailView):
+class ProductFormView(
+    CommonPageMixin, BaseBreadcrumbMixin, CategoryBreadcrumbMixin, DetailView
+):
     template_name = "pages/product/form.html"
     model = Product
     breadcrumb_use_pk = False
     no_list = True
 
+    def page_title(self):
+        return " ".join(self.object.name, _("Formulier"))
```

Comment:
I assume you want an f-string here or something like this. We will get an error in join.

Reply:
Yes, you are right. I added a commit to fix this.